src/utils/bounties.ts

```diff
@@ -1,25 +1,29 @@
 import { get } from 'svelte/store';
 import { activeAccount, activeAccountBounties, bounties } from '../stores';
 import { type Bounty } from '../types/bounty';
+import { isCurator } from './isCurator';
 
 /**
  * Sets the store's `activeAccountBounties` value to filtered bounties
  * that are related to the `activeAccount` account only.
  */
 export function setActiveAccountBounties() {
 	const allBounties: Bounty[] = get(bounties);
-	const address = get(activeAccount)?.address;
-	if (!address) {
+	const account = get(activeAccount);
+	if (!account) {
 		return;
 	}
+	const address = account.address;
 
 	const filteredBounties: Bounty[] = [];
 	for (const bounty of allBounties) {
-		if (bounty.proposer === address || bounty.curator === address) {
+		if (bounty.proposer === address || isCurator(bounty)) {
 			filteredBounties.push(bounty);
 			continue;
 		}
-		const filteredChildBounties = bounty.childBounties.filter(({ curator }) => curator === address);
+		const filteredChildBounties = bounty.childBounties.filter((childBounty) =>
+			isCurator(childBounty)
+		);
```

Comment:
```suggestion
		const filteredChildBounties = bounty.childBounties.filter(isCurator);
```
Would this work even better?